pages/test_analytics/js_collectors.md.erb

```diff
@@ -0,0 +1,55 @@
+# Ruby Collectors
+
+The beta release of Test Analytics ships with a Jest test collector.
+You you can also upload test results by importing [JSON](/docs/test-analytics/importing-json) or [JUnit XML](/docs/test-analytics/importing-junit-xml).
```

Comment:
```suggestion
You can also upload test results by importing [JSON](/docs/test-analytics/importing-json) or [JUnit XML](/docs/test-analytics/importing-junit-xml).
```

Reply:
I feel like this should crop up in a decent linter as well. Might be the different case that is preventing that?